samples/cmake/enet/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+if(NXDK)
+  # Hack for winsock2.h and mmsystem.h
+  include_directories(SYSTEM "${NXDK_DIR}/lib/winapi/ws2_32;${NXDK_DIR}/lib/winapi/winmm")
+
+  # Create lwip import lib
+  add_subdirectory(${NXDK_DIR}/lib/net/lwip "${CMAKE_BINARY_DIR}/lwip/")
+  get_target_property(LWIPCORE_INCLUDES lwipcore INCLUDE_DIRECTORIES)
+  message("${LWIPCORE_INCLUDES}")
+  set_target_properties(lwipcore
+                        PROPERTIES INCLUDE_DIRECTORIES "${NXDK_DIR}/lib/net/lwip/src/include/;${NXDK_DIR}/lib/net/nforceif/include/")
+  set_target_properties(lwipcore
+                        PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${NXDK_DIR}/lib/net/lwip/src/include/;${NXDK_DIR}/lib/net/nforceif/include/")
+  get_target_property(LWIPCORE_INCLUDES lwipcore INCLUDE_DIRECTORIES)
+  message("${LWIPCORE_INCLUDES}")
+
+  # Add pktdrv
+  add_library(pktdrv ${NXDK_DIR}/lib/net/pktdrv/pktdrv.c)
+  target_include_directories(pktdrv PUBLIC ${NXDK_DIR}/lib/net/pktdrv/)
+
+  # Combine lwip and Xbox driver into libnet
+  add_library(net
+    "${NXDK_DIR}/lib/net/nforceif/src/driver.c"
+    "${NXDK_DIR}/lib/net/nforceif/src/sys_arch.c"
+  )
+  target_link_libraries(net lwipcore pktdrv)
+endif()
```

Comment:
This should be part of nxdk, not of this sample.

Reply:
@teufelchen1: This is probably a good task for you
Maybe @kosmas12 if @teufelchen1 doesn't want to do this.

(nxdk issue 454: "Network stack should be a static library")